trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.test.ts

```diff
@@ -147,13 +159,61 @@ describe('controllers/utilisation-reports/utilisation-report-reconciliation-for-
         facilityId: premiumPaymentsFacilityId,
       };
 
+      const paymentDetailsFilters = {
+        facilityId: paymentDetailsFacilityId,
+        paymentCurrency: paymentDetailsPaymentCurrency,
+        paymentReference: paymentDetailsPaymentReference,
+      };
+
+      const paymentDetailsFiltersViewModel = {
```

Comment:
Why do we need to do this please, the controller simply calls `mapPaymentDetailsFiltersToPaymentDetailsFiltersViewModel`, but here we are duplicating what `mapPaymentDetailsFiltersToPaymentDetailsFiltersViewModel` does. Can we just call `mapPaymentDetailsFiltersToPaymentDetailsFiltersViewModel` instead?